simapp/test_helpers.go

```diff
@@ -50,6 +55,17 @@ var DefaultConsensusParams = &abci.ConsensusParams{
 	},
 }
 
+// SetupOptions defines arguments that are passed into `Simapp` constructor.
+type SetupOptions struct {
```

Comment:
Is there default setup options?

Reply:
No. I think we can use `simapp.Setup` for default options.